dynamixel_workbench_controllers/src/dynamixel_workbench_controllers.cpp

```diff
@@ -134,7 +139,7 @@ bool DynamixelController::initDynamixels(void)
     {
       if (dxl.first == info.first)
       {
-        if (info.second.item_name != "ID" && info.second.item_name != "Baud_Rate")
+        if (info.second.item_name != "ID" && info.second.item_name != "Baud_Rate" && info.second.item_name != "Torque_Enable")
```

Comment:
Added additional conditional check:
info.second.item_name != "Torque_Enable"

If Torque_Enable accidentally set from yaml file such as:
pan:
    ID:1
    Torque_Enable:1
    Return_Delay_Time:0
tilt:
    ID:2
    Torque_Enable:1
    Return_Delay_Time:0

This conditional check prevents the reported error.